### PR TITLE
Setup auto formatting

### PR DESCRIPTION
**Pull request by gstvribs**
#4

### CODE REVIEW

File: packages/client/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "bracketSpacing": true,
+    "semi": false,
```
Comment (epsxy) [inline packages/client/.prettierrc]:
```suggestion
    "semi": true,
```

To be consistent with server rules.

File: packages/client/.prettierrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    "singleQuote": true,
+    "trailingComma": "es5",
```
Comment (epsxy) [inline packages/client/.prettierrc]:
```suggestion
    "trailingComma": "all",
```

**Comment by epsxy** [comment]
@gstvribs thanks for this contribution and the work done :smiley: . 
What do you think of my suggestions?

**Comment by gstvribs** [comment]
> @gstvribs thanks for this contribution and the work done 😃 . I think it would be great to have some consistency regarding the prettier rule throught the app.
> What do you think of my suggestions?

Thanks, that suggestions was nice, thanks for that : )

**Comment by epsxy** [comment]
@gstvribs Great! Many thanks for you nice contributions :tada: :+1: .

**Pull request merged**